src/io/flutter/run/bazelTest/BazelTestLaunchState.java

```diff
@@ -67,4 +91,61 @@ public static BazelTestLaunchState create(@NotNull ExecutionEnvironment env, @No
     }
     return launcher;
   }
+
+  @Nullable
+  @Override
+  protected ConsoleView createConsole(@NotNull Executor executor) throws ExecutionException {
+    // If the --machine output flag is not turned on, then don't activate the new window.
+    if (fields.getAdditionalArgs() == null || !fields.getAdditionalArgs().contains("--machine")) {
```

Comment:
A nice to have eventually might be to have all the flags be defined in one place.

Reply:
Interesting idea, I think it's worth trying out.  Introduced with documentation in a new Flags class.